packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6040,6 +6065,20 @@ if(devices.real().deviceType === 'desktop') {
                 helper.checkAttributes($listItemContainer, {});
             });
         });
+
+        ['disabled', 'readOnly'].forEach(disabledOption => {
+            QUnit.test(`Input aria-autocomplete is none if ${disabledOption} is true`, function(assert) {
```

Comment:
let's also add a tests checking that the attribute is "none" if some of these options is set to `true` on init?

Reply:
Added